### DISCUSSION TITLE
failed to fetch items from kvstore

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#41298] [Alerting]
This type of messages are filling up our logs 

t=2021-11-03T23:59:17+0100 lvl=eror msg="failed to fetch items from kvstore" logger=ngalert.multiorg.alertmanager err="Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'key LIKE ?)' at line 1" namespace=alertmanager key=silences

we are running 8.2.3 it was the same with 8.2.2

**Comment by JohnnyQQQQ** [comment]
This was fixed in https://github.com/grafana/grafana/pull/40827 and will be shipped in the next version that is not a security fix release (like 8.2.3).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
